src/challenges/challengeTwo/IntegrationComponent/IntegrationComponent.test.tsx

```diff
@@ -1,8 +1,21 @@
-// import React from "react";
-// import { render, screen } from "@testing-library/react";
-// import userEvent from "@testing-library/user-event";
-// import { IntegrationComponent } from "./IntegrationComponent";
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { IntegrationComponent } from "./IntegrationComponent";
 
-// describe("IntegrationComponent", () => {
-//   it("renders...", () => {});
-// });
+describe("IntegrationComponent", () => {
+  it("opens the form modal and submits the form", () => {
```

Comment:
"allows the user to open the modal, fill in the form, and submit successfully" is a much better description. 

You might have wanted to separate your tests too.